babylon.yaml

```diff
@@ -0,0 +1,13 @@
+babylon_binary: /usr/local/bin/bbi
```

Comment:
Was committing this file (babylon.yaml) unintentional?

Reply:
yep. Good catch.